packages/core/redux/bitcoin/thunks.js

```diff
@@ -0,0 +1,212 @@
+      accept: acceptTransaction,
+      reject: rejectTransaction,
+    },
+  }))
+}
+
+const acceptTransaction = (entry) => async (dispatch, getState) => {
+  dispatch(BitcoinActions.acceptTransaction(entry))
+
+  const state = getState()
+  const signer = getSigner(state)
+
+  const selectedEntry = pendingEntrySelector(entry.tx.from, entry.key, entry.blockchain)(getState())
+
+  if (!selectedEntry) {
+    // eslint-disable-next-line no-console
+    console.error('entry is null', entry)
+    return // stop execute
+  }
+
+  return dispatch(processTransaction({
+    entry: selectedEntry,
+    signer,
+  }))
+}
+
+const rejectTransaction = (entry) => (dispatch) => dispatch(BitcoinActions.rejectTransaction(entry))
+
+const processTransaction = ({ entry, signer }) => async (dispatch, getState) => {
+  await dispatch(signTransaction({ entry, signer }))
+  const signedEntry = pendingEntrySelector(entry.tx.from, entry.key, entry.blockchain)(getState())
+  if (!signedEntry) {
+    // eslint-disable-next-line no-console
+    console.error('signedEntry is null', entry)
+    return null // stop execute
+  }
+  return dispatch(sendSignedTransaction({
+    entry: signedEntry,
+  }))
+}
+
+const signTransaction = ({ entry, signer }) => async (dispatch, getState) => {
+  try {
+    const { tx } = entry
+    const txb = tx.prepared
+    const signInputs = BitcoinUtils.signInputsMap[entry.blockchain]
+    const state = getState()
+
+    if (typeof signInputs === 'function') {
+      const network = getSelectedNetwork()(state)
+      const pk = signer.privateKey.substring(2, 66) // remove 0x
+      const bitcoinNetwork = bitcoin.networks[network[entry.blockchain]]
+      const bitcoinSigner = BitcoinUtils.createBitcoinWalletFromPK(pk, bitcoinNetwork)
+      signInputs(txb, tx.inputs, bitcoinSigner)
+    }
+
+    dispatch(BitcoinActions.bitcoinTxUpdate(BitcoinUtils.createBitcoinTxEntryModel({
+      ...entry,
+      tx: {
+        ...entry.tx,
+        signed: txb.build(),
+      },
+    })))
+  } catch (error) {
+    dispatch(BitcoinActions.bitcoinTxUpdate(BitcoinUtils.createBitcoinTxEntryModel({
+      ...entry,
+      isErrored: true,
+      error,
+    })))
+    throw error
+  }
+}
+
+const sendSignedTransaction = ({ entry }) => async (dispatch, getState) => {
+  dispatch(BitcoinActions.bitcoinTxUpdate(BitcoinUtils.createBitcoinTxEntryModel({
+    ...entry,
+    isPending: true,
+  })))
+
+  // eslint-disable-next-line
+  entry = pendingEntrySelector(entry.tx.from, entry.key, entry.blockchain)(getState())
+  if (!entry) {
+    // eslint-disable-next-line no-console
+    console.error('entry is null', entry)
+    return // stop execute
+  }
+
+  const node = BitcoinUtils.getNodeByBlockchain(entry.blockchain)
+  if (!node) {
+    dispatch(BitcoinActions.bitcoinTxUpdate(BitcoinUtils.createBitcoinTxEntryModel({
+      ...entry,
+      isErrored: true,
+    })))
+    return null
+  }
+
+  const res = await node.send(entry.tx.from, entry.tx.signed.toHex())
```

Comment:
Concept of 'nodes' is deprecated. Please create new module packages/core/redux/bitcoin/http.js
as it was done for profile 

Http requests module (axios): https://github.com/ChronoBank/ChronoMint/blob/develop/packages/core/redux/profile/service.js

Usage example:
https://github.com/ChronoBank/ChronoMint/blob/develop/packages/core/redux/profile/thunks.js